README.rst

```diff
@@ -13,6 +13,26 @@ Documentation  https://DiamondLightSource.github.io/dodal
 Releases       https://github.com/DiamondLightSource/dodal/releases
 ============== ==============================================================
 
+Testing Connectivity
+--------------------
+
+You can test your connection to a beamline if it's PVs are visible to your machine with:
+
+.. code:: shell
+
+    # On any workstation:
+    dodal connect <BEAMLINE>
+
+    # On a beamline workstation, this should suffice:
+    dodal connect ${BEAMLINE}
```

Comment:
Should: Sorry to scope creep a bit can we add this to the PR template while it's not in CI. As a reviewer it's basically the first thing I run and a lot of the time it fails.